Dshop/apps/users/templates/users/registration.html

```diff
@@ -0,0 +1,39 @@
+{% extends 'base.html' %}
+{% load static %}
+{% block stripe %}
+    <script src="https://js.stripe.com/v3/"></script>
+    <script src="{% static 'main.js' %}"></script>
+    <script defer src="https://use.fontawesome.com/releases/v6.4.0/js/all.js"></script>
+{% endblock %}
+
+{% block content %}
+    <div class="container">
+        <div class="d-flex justify-content-center">
+            <div class="mx-auto col-10 col-md-8 col-lg-6 vh-100 mt-lg-3">
+                <form method="POST" action="">
+                    {% csrf_token %}
+                    <div class="form-group mb-3 col-lg-7">
+                        <label for="username" class="mb-2">{{ form.username.name.capitalize}}</label>
+                        {{form.username}}
+                    </div>
+                    <div class="form-group mb-3 col-lg-7">
+                        <label for="username" class="mb-2">Password</label>
+                        {{form.password1}}
+                    </div>
+                    <div class="form-group mb-3 col-lg-7">
+                        <label for="username" class="mb-2">Confirm Password</label>
+                        {{form.password2}}
+                    </div>
+                    <div class="form-group mb-3 col-lg-7">
+                        <label for="username" class="mb-2">Email</label>
```

Comment:
Field is email, label for username - please fix.

Reply:
Actually those labels for attribute is wrong in every template. Please go over and fix all of them. 
Try this:
https://docs.djangoproject.com/en/dev/topics/forms/#looping-over-the-form-s-fields
mainly: {{ field.label_tag }}